docs/Docker.md

```diff
@@ -156,3 +156,58 @@ git push --set-upstream origin main
 
 A Docker container can be used to investigate a bug, or to develop a new feature. Some like to create a new images for each new version of ONNX-MLIR; others prefer to create one image and use git to update the main branch and use git to switch between multiple branches. Both are valid approaches.
 
+## Using a devcontainer
+Another way of building onnx-mlir for development in VSCode is using a devcontainer. This way you only mount your source folder, meaning that changes you do are saved on your local machine. For this setup to work you need a `Dockerfile` and a `devcontainer.json` file. Both are provided in `docs/devcontainer-example`. 
+The Dockerfile is a simple Dockerfile based on the precompiled LLVM/MLIR image that is shared. It looks like this:
+
+```docker
```

Comment:
You should probably link to the file instead of showing the contents here. This will get out of sync very quickly